cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +200,92 @@
 
+# IMPORTANT NOTE: the support of the ACME protocol and automatic renewal of
+# certificates is currently in the EXPERIMENTAL stage.  Once we have more
+# experience with people using it out in the wild, we will gradually move it to
+# the PRODUCTION stage.
+#
+# Configure ACME, which allows the webpkgserver to automatically renew Signed
+# Exchange certificates.
+#
+# ACME is a protocol that allows automatic certificate renewal. Setting the
+# Enable parameter to true will make webpkgserver automatically request a
+# renewed certificate whenever it has determined that the current certificate
+# is expired or about to expire.
+#
+# If you set up more than one instance of webpkgserver, the recommended practice
+# is to have only one instance handle automatic certificate renewals and let
+# other instances reload the fresh certificate from the disk. It allows
+# certificates to be shared on all webpkgserver instances (which may help
+# minimize the monetary cost, subject to Terms of Service set by your
+# Certificate Authority) and prevents webpkgserver from bombarding your
+# Certificate Authority with certificate requests.
+#
+# webpkgserver uses the ACME library from github.com/go-acme/lego.
+# 
+# For the full ACME spec, see:
+#   https://tools.ietf.org/html/rfc8555 
+[SXG.ACME]
+  # Enable webpkgserver to attempt to auto renew certificates using ACME.
+  #Enable = false 
+
+  # The ACME discovery URL. It is specified by the Certificate Authority that
+  # doles out your certificates. As of August 2020, only DigiCert supports
+  # automatic renewals of signed exchange certificate:
+  # https://docs.digicert.com/certificate-tools/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/
+  # Required when ACME is enabled.
+  DiscoveryURL = '<Your Discovery URL>'
+
+  # The email address registered to the Certificate Authority for your signed
+  # exchange certificates.
+  # Required when ACME is enabled.
+  Email = 'user@example.com'
+
+  # For the remaining configuration items, it is important to understand the
+  # different challenges employed as part of the ACME protocol:
+  # https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html#identifier-validation-challenges
+  # https://letsencrypt.org/docs/challenge-types/
+  # https://certbot.eff.org/docs/challenges.html/
+  #
+  # Note that you do not need to set the fields for all of these challenges. It
+  # is typically sufficient to have a setting for just one of the challenges. If
+  # more than one method is configured, the go-acme/lego library will decide
+  # pick one of them for primary use and use other settings as backup. For
+  # wildcard certificates, nevertheless, DNSProvider is the only supported
+  # method of validation, and others cannot be used. See DNSProvider for more
+  # detail.
+
+  # The http server root directory where the ACME http challenge token should
+  # be deposited. Note that you may need configure your reverse-proxy server to 
```

Comment:
Thanks, it's become a bit clearer, but I still don't understand why and how the user needs to configure their reverse-proxy server when running multiple webpkgserver instances.

Reply:
Where does it say that the reverse-proxy configuration is related to the multiple webpkgserver instance? Because they are totally not related.

Reply:
Ok changed it some more, let me know if that's ok.